### PR TITLE
fix: security vulnerabilities

### LINKED ISSUE #77: [Secutirty] Vulnerability in Mocha.js
#### Description
Hello folks,

I'm working on a project where we use `converter-js`, and our monitoring tools caught that there are some vulnerabilities in Mocha, up until the latest major release.
I've checked the [breaking changes](https://github.com/mochajs/mocha/releases?q=breaking&expanded=true) and it looks like it won't be a problem to bump the version directly from `^6.1.4` to `^9.1.3` (the tests are passing locally as well).
The only concerning thing seems to be the drop of Node.js v10.x support, however the vulnerability is not fully addressed in the latest v8 release.

#### Details
The particular vulnerability is [CVE-2021-3807](https://www.whitesourcesoftware.com/vulnerability-database/CVE-2021-3807), and it's part of `ansi-regex`, which is a transitive dependency of `mocha`. Here are the dependency graphs:

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Fix security vulnerabilities:
- move `mocha` package to the `devDependencies`
- update packages to fix security vulnerabilities. Some vulnerabilities are still (only moderate, not high), even after several `npm run audit`:

```
# npm audit report

ansi-regex  >2.1.1 <5.0.1
Severity: moderate
 Inefficient Regular Expression Complexity in chalk/ansi-regex - https://github.com/advisories/GHSA-93q8-gq69-wqmw
fix available via `npm audit fix`
node_modules/npm/node_modules/cli-table3/node_modules/ansi-regex
node_modules/npm/node_modules/string-width/node_modules/ansi-regex
  strip-ansi  4.0.0 - 5.2.0
  Depends on vulnerable versions of ansi-regex
  node_modules/npm/node_modules/string-width/node_modules/strip-ansi
    string-width  2.1.0 - 4.1.0
    Depends on vulnerable versions of strip-ansi
    node_modules/npm/node_modules/string-width

json-schema  <0.4.0
Severity: moderate
json-schema is vulnerable to Prototype Pollution - https://github.com/advisories/GHSA-896r-f27r-55mw
fix available via `npm audit fix`
node_modules/npm/node_modules/json-schema
  jsprim  0.3.0 - 1.4.1 || 2.0.0 - 2.0.1
  Depends on vulnerable versions of json-schema
  node_modules/npm/node_modules/jsprim

5 moderate severity vulnerabilities
```

Also update default version to convert from `2.1.0` to `2.2.0`.

**Related issue(s)**
Resolves #77

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ if (!asyncapiFile) {
   program.help(); // This exits the process
 }
 if (!version) {
-  version = '2.1.0';
+  version = '2.2.0';
```
Comment (fmvilas) [inline cli.js:35]: Is this related to fixing security vulnerabilities? 🤔

Comment (magicmatatjahu) [reply cli.js:35]: No but while testing cli I noticed that without specifying the version it converts the spec to 2.1.0 and not to the latest version 2.2.0. We overlooked this in the previous release :)

Comment (fmvilas) [reply cli.js:35]: Oh ok. I'd say you should be opening another PR for that change but I'm not going to block you for that. Just please take this into account for future PRs. If we had to revert this PR for some reason, we could forget about keeping this change that's not related to the purpose of the PR.

**Review by fmvilas — APPROVED** [review]

File: cli.js (hunk 1 above)
Comment (magicmatatjahu) [reply cli.js]: Right, I will open another PR with that. Could you accept again? :) Thanks!

**Review by fmvilas — APPROVED** [review]

**Pull request merged**